core/app/[locale]/(default)/(faceted)/category/[slug]/page-data.ts

```diff
@@ -6,6 +7,8 @@ import { graphql, VariablesOf } from '~/client/graphql';
 import { revalidate } from '~/client/revalidate-target';
 import { BreadcrumbsFragment } from '~/components/breadcrumbs/fragment';
 
+import { WishlistSheetFragment } from '../../../../../../components/wishlist-sheet/fragment';
```

Comment:
If it's possible maybe it would look prettier to use:

```
import { WishlistSheetFragment } from '~/components/wishlist-sheet/fragment';
```

that looks more consistent with line above 💅

Reply:
for sure, I should rely less on import autocomplete 😁